spec/fixtures/rails42.rb

```diff
@@ -10,6 +10,7 @@ class Dummy < Rails::Application
   config.secret_key_base = '2624599ca9ab3cf3823626240138a128118a87683bf03ab8f155844c33b3cd8cbbfa3ef5e29db6f5bd182f8bd4776209d9577cfb46ac51bfd232b00ab0136b24'
   config.session_store :cookie_store, key: '_rails32_session'
 
+  config.log_level = :info
```

Comment:
🙋 Is this needed?

Reply:
Rails 4.2 will raise a warning when `log_level` isn't specified in production 🍦